### PR TITLE
Update two-data-centers-in-one-city-deployment.md

### PR DESCRIPTION
**Pull request by nickzabc**
1、修改推荐副本数为6。
2、AZ2的角色从 “Voter”改为“Follower”。

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Close https://github.com/pingcap/docs-cn/issues/12135
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@nolouch PTAL

**Review by nolouch — APPROVED** [review]
lgtm

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: two-data-centers-in-one-city-deployment.md [hunk 1]
```diff
@@ -83,7 +89,7 @@ alertmanager_servers:
 
 ### Placement Rules 规划
 
-为了按照规划的集群拓扑进行部署，你需要使用 [Placement Rules](/configure-placement-rules.md) 来规划集群副本的放置位置。以 4 副本（2 个 Voter 副本在主 AZ，1 个 Voter 副本和 1 个 Learner 副本在从 AZ）的部署方式为例，可使用 Placement Rules 进行如下副本配置：
+为了按照规划的集群拓扑进行部署，你需要使用 [Placement Rules](/configure-placement-rules.md) 来规划集群副本的放置位置。以 6 副本（3 个 Voter 副本在主 AZ，2 个 Voter 副本和 1 个 Learner 副本在从 AZ）的部署方式为例，可使用 Placement Rules 进行如下副本配置：
```
Comment (Oreoxmt) [inline two-data-centers-in-one-city-deployment.md:92]:
```suggestion
为了按照规划的集群拓扑进行部署，你需要使用 [Placement Rules](/configure-placement-rules.md) 来规划集群副本的放置位置。以 6 副本（3 个 Voter 副本在主 AZ，2 个 Follower 副本和 1 个 Learner 副本在从 AZ）的部署方式为例，可使用 Placement Rules 进行如下副本配置：
```
@nickzabc PTAL

File: two-data-centers-in-one-city-deployment.md [hunk 2]
```diff
@@ -69,10 +69,16 @@ tikv_servers:
...
     config:
-      server.labels: { az: "west", rack: "west-1", host: "32" }
+      server.labels: { az: "east", rack: "east-3", host: "32" }
   - host: 10.63.10.33
     config:
-      server.labels: { az: "west", rack: "west-2", host: "33" }
+      server.labels: { az: "west", rack: "west-1", host: "33" }
+  - host: 10.63.10.34
+    config:
+      server.labels: { az: "west", rack: "west-2", host: "34" }
```
Comment (Oreoxmt) [inline two-data-centers-in-one-city-deployment.md:78]:
```suggestion
      server.labels: { az: "west", rack: "west-2", host: "34" }
```

**Pull request merged**